tests/populator_test.py

```diff
@@ -120,7 +120,7 @@ def test_get_helper(self, *args):
         """Test get_device_helper for dm devices."""
         device_is_dm = args[0]
         device_is_dm_lvm = args[6]
-        data = Mock()
+        data = {'SYS_PATH': 'dummy'}
```

Comment:
I have. Local tests are passing. The return value of the affected function is mocked in all cases. This is just to prevent KeyError since that function is actually being executed before the mock happens.